wordpress-mobile/WordPress-iOS
# Feature/login/epilogue list

Gives the login epilogue screen the visual design polish.

![epilogue-shot-for-pr](https://cloud.githubusercontent.com/assets/517257/25869733/a556729c-34be-11e7-90d6-291078d5c701.png)

To test:
_Note: The new login screen only appears when the app has no accounts (not when clicking to add a new site)_
1. Ensure that login is still working 😉
2. Check the look of the epilogue when using an account with just one site
3. Check with many sites

### Missing features
- Epilogue not appearing for self-hosted logins, but it will in a future PR
- A "Connect another site" button hasn't been added yet, but will in a future PR

Needs review: @elibud 
_preemptive code review request_

Also: @mattmiklic can you verify as well?
